com.dosbox_x.DOSBox-X.yaml

```diff
@@ -0,0 +1,58 @@
+app-id: com.dosbox_x.DOSBox-X
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+
+command: dosbox-x
+
+finish-args:	# flatpak permissions
+  - --device=all	# needed for gamepads and serial/parallel
+  - --share=ipc	        # needed for X11
+  - --share=network	# needed for networking (NE2000, IPX)
+  - --socket=x11
```

Comment:
Doesn't it work natively on Wayland due to SDL2?

Reply:
Just tested and it seems fine, should be replaced with `--socket=wayland` and `--socket=fallback-x11`.

Reply:
Unfortunately it is not fine. I tested it here on Fedora 33 beta and launching with wayland causes the window decorations and controls to vanish. In addition the window is the wrong size and the mouse location tracking gets messed up. I think the later two issues are due to the fact that the title bar is missing and the window gets stretched to the window+title-bar which in turn causes the mouse position to get messed up.

Reply:
This issue is also not specific to flatpak. The same thing happens when running the binary normally using
```
SDL_VIDEODRIVER=wayland dosbox-x
```

Reply:
Ack, please ignore my comment.